.github/workflows/python-poetry-release.yaml

```diff
@@ -94,13 +94,15 @@ jobs:
           fetch-release-information: "true"
 
       - name: Commit and push pyproject.toml file
-        uses: bakdata/ci-templates/actions/commit-and-push@v1.0.0
+        uses: bakdata/ci-templates/actions/commit-and-push@v1.25.0
```

Comment:
we removed the leading `v` in tags, remember? 😉 
```suggestion
        uses: bakdata/ci-templates/actions/commit-and-push@1.25.0
```

Reply:
not for that version xD

Reply:
oh right! 😄